odoo/tools/misc.py

```diff
@@ -1256,6 +1256,14 @@ def format_time(env, value, tz=False, time_format='medium', lang_code=False):
 
     return babel.dates.format_time(value, format=time_format, locale=locale)
 
+
+def _format_time_ago(env, time_delta, lang_code=False):
+    if not lang_code:
+        lang_code = env.context.get('lang') or env['res.lang'].get_installed()[0][0]
```

Comment:
This can work if you are certain the language in the context is a language installed on the system. In #35633 the issue is that `formatLang` was used with language of the user's browser (which is sometimes exotic like `es_419`). Calling `babel.Locale.parse` with a language unknown by the system will crash. I haven't checked a lot the usage but as it is in website, it looks fishy.

Reply:
you're absolutely right, I got the point.